kuma/core/middleware.py

```diff
@@ -309,3 +316,15 @@ def process_request(self, request):
                 urljoin(settings.SITE_URL, request.get_full_path())
             )
         return None
+
+
+if settings.DJANGO_1_10:
+    class RedirectsMiddleware(MiddlewareMixin, OriginalRedirectsMiddleware):
+        """
+        Enables the redirect_urls middleware to be used with both MIDDLEWARE
+        and MIDDLEWARE_CLASSES until a newer version of django-redirect-urls
```

Comment:
``django-redirect-urls`` is https://github.com/pmac/django-redirect-urls, we should submit a PR.